iml-agent/Cargo.toml

```diff
@@ -41,6 +42,7 @@ iml-wire-types = { path = "../iml-wire-types", version = "0.2" }
 liblustreapi = { path = "../liblustreapi", version = "0.1" }
 iml-fs = { path = "../iml-fs", version = "0.1.0" }
 iml-util = { path = "../iml-util", version = "0.1.0" }
+tempfile = "3.1.0"
```

Comment:
Since it is used in tests only, it should go to [dev-dependencies].

Reply:
Done